schema/home/home_page_artist_module.js

```diff
@@ -57,27 +65,31 @@ export const HomePageArtistModuleType = new GraphQLObjectType({
       },
     },
     key: {
+      description: 'Module identifier.',
       type: GraphQLString,
     },
     results: {
       type: new GraphQLList(Artist.type),
       resolve: ({ key, display, params }, options, { rootValue: { accessToken, userID } }) => {
-        return Results[key].resolve(accessToken, userID);
+        return HomePageArtistModuleTypes[key].resolve(accessToken, userID);
       },
     },
   },
 });
 
 const HomePageArtistModule = {
   type: HomePageArtistModuleType,
-  description: 'Single artist module to show on the home screen',
+  description: 'Single artist module to show on the home screen.',
   args: {
     key: {
-      type: GraphQLString,
-      description: 'Module key',
+      description: 'Module identifier.',
+      type: new GraphQLEnumType({
+        name: 'HomePageArtistModuleTypes',
+        values: HomePageArtistModuleTypes,
```

Comment:
Surprised this works, maybe more intentional to do something like `keys(HomePageArtistModuleTypes)`

https://lodash.com/docs#keys